### ISSUE TITLE
Can't save file

### ISSUE THREAD

**Issue by ANtlord** [flathub/com.github.phase1geo.minder#3]
Hello!

So here is the subject. I try to save a document to an arbitrary folder with my project but an expected file doesn't appear in the designated folder. I installed the application from a repository of Fedora so it doesn't have the issue. I have no idea on troubleshooting flatpak packages so it would be great to have some instructions.

Technical info
```
➜  / flatpak info com.github.phase1geo.minder

Minder - Create, develop and visualize your ideas

          ID: com.github.phase1geo.minder
         Ref: app/com.github.phase1geo.minder/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.11.3
     License: GPL-3.0+
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 102.4 MB
     Runtime: org.freedesktop.Platform/x86_64/20.08
         Sdk: org.freedesktop.Sdk/x86_64/20.08

      Commit: 14d39a1759a0aae22f8da1938d56db71da0bdf73af21662377d4d33cb83e2d72
      Parent: bc1baca86f5e674b1a356d90605f537e2c49e25e2c9c8521ae4aa8f14e8fabe7
     Subject: build: unbundle Vala (494b90c4)
        Date: 2020-10-26 10:35:35 +0000
```
OS: Fedora 32

I'm happy to provide any additional technical informtaion

**Comment by tim77** [comment]
Hello @ANtlord

It feels like the same issue? https://github.com/phase1geo/Minder/issues/191

**Comment by ANtlord** [comment]
I suppose so... should I close it and provide my system information there?

**Comment by tim77** [comment]
> should I close my issue

No, thanks for reporting. And this is proper place for this issue since non-flatpak version doesn't suffer from this. We need to gather all information altogether and steps how to reproduce it and file a bug in `xdg-desktop-portal`...

**Comment by ANtlord** [comment]
$ flatpak --version
Flatpak 1.8.3
$ rpm -q xdg-desktop-portal
xdg-desktop-portal-1.8.0-1.fc32.x86_64
$ hostnamectl
   Static hostname: localhost.localdomain
         Icon name: computer-desktop
           Chassis: desktop
        Machine ID: 35722aea6c5d46879f241466669ada3d
           Boot ID: a445ce6540ae4859b945299517175be8
  Operating System: Fedora 32 (Workstation Edition)
       CPE OS Name: cpe:/o:fedoraproject:fedora:32
            Kernel: Linux 5.9.14-100.fc32.x86_64
      Architecture: x86-64

Steps to reproduce
1. Install Minder
2. Create a document
3. Save the document in a home directory
4. In the list of files of the home directory no file of the document

**Comment by daudtivan** [comment]
Exact same thing here on Elementary OS 5.1

**Comment by daudtivan** [comment]
> Exact same thing here on elementary OS 5.1.7 Hera Linux 4.15.0-162-generic GTK 3.22.30 Flatpak 1.8.3

Just found saved files on "/run/user/1000/doc/by-app/com.github.phase1geo.minder/21c5523f"
There are several directories that seems to be app instances/sessions with other attempts.
Unfortunately I can't open these files. Nothing happens.

Hope it helps.

**Comment by daudtivan** [comment]
Problem solved by installing Flatseal, as indicated above, and adjusting permission to access my home dir.
Moved above files to my home dir and everything is working as expected now.

This might hv to do with flatpak packing script/manifest generation. These permissions should be indicated somewhere there.
My $0,02

**Comment by tim77** [comment]
Added `filesystem=home` as workaround since there is no progress with proper fix in portals.
